src/it/scala-2.12/com/linagora/gatling/imap/scenario/it/ImapAuthenticationScenarioIT.scala

```diff
@@ -26,151 +26,158 @@ abstract class BaseIt(server: RunningServer) extends GatlingFunSpec {
   }
 }
 
-
-class ImapAuthenticationScenarioIT extends BaseIt(CyrusServer.start()) {
```

Comment:
You don't need to comment all scenarios to just run one test. You can use in sbt:

```
GatlingIt/testOnly FQDN
```

with I guess your FQDN being here `com.linagora.gatling.imap.scenario.MassiveOperationScenario` :)

Reply:
ah or is it for the CI?

Reply:
Yeh, only for the CI